### PR TITLE
Updated Screenshot Tests and fixed an issue with the MockMediaProvider

### PR DESCRIPTION
**Pull request by Velin92**
Updated all the screenshots to adapt to the new designs.

Also fixed an issue that did not fetch an image for the MockMediaProvider to render mocked avatar rooms, and an issue where the roomID was incorrectly using the id of the proxy and not the roomId coming from the timeline

### CODE REVIEW

File: ElementX/Sources/UITests/UITestsAppCoordinator.swift [hunk 1]
```diff
@@ -275,10 +275,29 @@ class MockScreen: Identifiable {
         case .roomDetailsScreen:
             let navigationStackCoordinator = NavigationStackCoordinator()
             let userNotificationController = UserNotificationController(rootCoordinator: navigationStackCoordinator)
+            var roomProxy = MockRoomProxy(
+                displayName: "Room",
+                isEncrypted: true,
+                members: [.mockAlice, .mockBob, .mockCharlie]
+            )
+            roomProxy.id = "MockRoomIdentifier"
```
Comment (pixlwave) [inline ElementX/Sources/UITests/UITestsAppCoordinator.swift:283]: Style and you can use the synthesised init for the `id`.
```suggestion
            var roomProxy = MockRoomProxy(id: "MockRoomIdentifier",
                                          displayName: "Room",
                                          isEncrypted: true,
                                          members: [.mockAlice, .mockBob, .mockCharlie])
```

File: UITests/Sources/RoomDetailsScreenUITests.swift [hunk 2]
```diff
@@ -22,6 +22,15 @@ class RoomDetailsScreenUITests: XCTestCase {
         let app = Application.launch()
         app.goToScreenWithIdentifier(.roomDetailsScreen)
         
+        XCTAssert(app.staticTexts["roomAvatarImage"].exists)
         app.assertScreenshot(.roomDetailsScreen)
     }
+
+    func testInitialStateComponentsWith() {
```
Comment (pixlwave) [inline UITests/Sources/RoomDetailsScreenUITests.swift:29]:
```suggestion
    func testInitialStateComponentsWithAvatar() {
```

**Review by pixlwave — APPROVED** [review]
Thanks for all these updates 👏

File: UITests/Sources/RoomDetailsScreenUITests.swift (hunk 2 above)
Comment (Velin92) [reply UITests/Sources/RoomDetailsScreenUITests.swift:29]: done

File: ElementX/Sources/UITests/UITestsAppCoordinator.swift (hunk 1 above)
Comment (Velin92) [reply ElementX/Sources/UITests/UITestsAppCoordinator.swift]: done

**Pull request merged**